scripts/beegfsmd.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+#
+disk_type=$1
+node_type=$2
+pools=$3
+MGMT_HOSTNAME=$4
+#
+BEEGFS_DISK=/mnt/beegfs
+BEEGFS_METADATA=${BEEGFS_DISK}/meta
+#
+yum install -y beegfs-meta
+sed -i 's|^storeMetaDirectory.*|storeMetaDirectory = '$BEEGFS_METADATA'|g' /etc/beegfs/beegfs-meta.conf
+sed -i 's/^sysMgmtdHost.*/sysMgmtdHost = '$MGMT_HOSTNAME'/g' /etc/beegfs/beegfs-meta.conf
+#
+sed -i 's/^connMaxInternodeNum.*/connMaxInternodeNum = 800/g' /etc/beegfs/beegfs-meta.conf
+sed -i 's/^tuneNumWorkers.*/tuneNumWorkers = 128/g' /etc/beegfs/beegfs-meta.conf
+#
+setup_data_disks()
```

Comment:
I did try and extract out creating the raid with the "create_raid0.sh" script.  This way the raid can be created first and 
lustre/beegfs only need to be told which device to use.  Does it make sense to use here?

Reply:
yes, I agree it would be good, to extract out the raid0 script and have that as a separate step (Maybe have a single raid0 script for lustre/beegfs). This current beegfs version determines the devices to be used (does not pass as arg in config.json), supports nvme, local ssd, managed disks and support for beegfs pools, so itwould take some time to refactor the code. So for now I would like to have a working version of beegfs available in the azurehpc repo, and make these changes in later versions.